docs/deploy_features.md

```diff
@@ -0,0 +1,58 @@
+
+# Enabling optional features
+
+This document covers optional features that can be enabled in the deployed Azure resources.
+You should typically enable these features before running `azd up`. Once you've set them, return to the [deployment steps](../README.md#deploying).
+
+* [Using GPT-4](#using-gpt-4)
+* [Enabling GPT-4 Turbo with Vision](#enabling-gpt-4-turbo-with-vision)
+* [Enabling Integrated Vectorization](#enabling-integrated-vectorization)
+* [Enabling authentication](#enabling-authentication)
+* [Enabling login and document level access control](#enabling-login-and-document-level-access-control)
+* [Enabling CORS for an alternate frontend](#enabling-cors-for-an-alternate-frontend)
+
+## Using GPT-4
+
+We generally find that most developers are able to get high quality answers using GPT 3.5. However, if you want to try GPT-4, you can do so by following these steps:
+
+* In `infra/main.bicep`, change `chatGptModelName` to 'gpt-4' instead of 'gpt-35-turbo'.
```

Comment:
Nice to have: it would be great to create and environment variable that the user can use to change the model instead of modifying the bicep directly

Reply:
I agree, I haven't figured out how to do that with the way we currently support OpenAI as well. Maybe if I stare hard enough I'll figure it out.

Reply:
Err actually I think I can do it, it'll just be a nested ternary. Doable though.